app.rb

```diff
@@ -1,23 +1,18 @@
-# pp [{ id: 1, hello: "World" }, 
-#     { id: 2, hello: "Ruby" }, 
-#     { id: 3, hello: "Moon" }, 
-#     { id: 4, hello: "Learner" }]
+pp [{ id: 1, hello: "World" },
+    { id: 2, hello: "Ruby" },
+    { id: 3, hello: "Moon" },
+    { id: 4, hello: "Learner" }]
 
-# p [1,2,3]
-# puts "Hello World!"
-# print "Pass this test, please."
-# p [1,2,3]
+p [{ id: 1, hello: "World" },
+{ id: 2, hello: "Ruby" },
+{ id: 3, hello: "Moon" },
+{ id: 4, hello: "Learner" }]
 
-# def add
-    
-# end
-# # 1 + 1
-# puts add + add
+p [1,2,3]
+pp [1, 2, 3]
+puts [1, 2, 3]
 
-def add num1, num2
-    puts num1
-    puts num2
-    num1 + num2
-end
+puts "Hello World!"
+print "Pass this test, please."
 
-puts add(1,3) + add(2,2)
\ No newline at end of file
+puts "Hi Steven"
```

Comment:
:(

Reply:
@stephenmckeon Different Steve, you wouldn't know him, he goes to a different company code bootcamp